clang/docs/ReleaseNotes.rst

```diff
@@ -630,6 +630,7 @@ Bug Fixes to C++ Support
 - Fix a bug on template partial specialization with issue on deduction of nontype template parameter
   whose type is `decltype(auto)`. Fixes (#GH68885).
 - Clang now correctly treats the noexcept-specifier of a friend function to be a complete-class context.
+- Fix crash caused by missing invocation of ``setInvalidDecl()`` for invalid ``Anon`` declarations. (#GH85447)
```

Comment:
Maybe "Fix an assertion failure when parsing an invalid members of an anonymous class" (the changelog is user facing)

Reply:
Oke

Reply:
Oke, thanks